softwareComponents/shapeReconfig/src/equality.cpp

```diff
@@ -0,0 +1,68 @@
+#include <shapeReconfig/equality.hpp>
+
+namespace rofi::shapereconfig {
+
+using namespace rofi::configuration;
+ 
+// Used for joint parameters, must be within tolerance (1/100 to recognize one degree at distance of 0.5)
+bool isClose( float f1, float f2 )
```

Comment:
why not add precision with default value?
also it's weird that the functions have different precision (so maybe a comment on that).

Reply:
I see...
You could add it as a template parameter - or pass in a lambda (something like `[](auto&& a, auto&& b){return isClose(a, b);}` when using it).

But I would probably just use a `constexpr auto precision = ...;` and use this.
So that at least if you want to change it, you can change it everywhere at one. If you think that each function should have a different precision, you can leave it, but I don't think that is the right choice.